CMakeLists.txt

```diff
@@ -118,7 +118,9 @@ endif()
 
 #Switch off modern thread local for dmlc-core, please see: https://github.com/dmlc/dmlc-core/issues/571#issuecomment-543467484
 add_definitions(-DDMLC_MODERN_THREAD_LOCAL=0)
-
+# disable stack trace in exception by default.
+add_definitions(-DDMLC_LOG_STACK_TRACE_SIZE=0)
```

Comment:
This changes the number of lines logged of a stack trace. Let's not change that in a minor release (1.6.1)

Reply:
Sure, updated

Reply:
(I assumed the same comment applied to the Makefile)